src/solver/mlir_common.cpp

```diff
@@ -58,6 +58,7 @@ static const char* DTypeName(miopenDataType_t ty)
     case miopenInt8: return "i8";
     case miopenFloat8: return "fp8";
     case miopenBFloat8: return "bfp8";
+    case miopenInt64: return "i64";
```

Comment:
Is there anybody familiar with MLIR? Just to double check that `i64` actually exists.

Reply:
@CAHEK7 Even if not, there should be no problem, this change is good.